examples/cemetery.ttl

```diff
@@ -90,6 +90,57 @@
            schema:ratingValue 2 ;
            schema:worstRating 1 .
 
+# Enrichment
+<Mention/1> a wev:Mention ;
```

Comment:
I was thinking of perhaphs using the [schema:Observation](https://schema.org/Observation) class but it seems to be extremely specific to statistical analyses and W3C cube. Still, worth using it maybe?

Reply:
schema:Observation is not adapted here, unless you want to consider smth like the coefficient of mentioning of Calais as variable you are measuring in your data sample ;). What you really want is representing annotations. 
We can either create our own (but not a wev:Metnion, rather wev:Annotation schema:additionalType wev:Mention  You can also use a specific ontology for this, such as NIF: https://persistence.uni-leipzig.org/nlp2rdf/ontologies/nif-core/nif-core.html

Reply:
NIF is indeed applied in the NLP world and this class seems to be what we need https://persistence.uni-leipzig.org/nlp2rdf/ontologies/nif-core/nif-core.html#d4e59